### PR TITLE
Theme and plugin updates

### PR DESCRIPTION
**Pull request by argenos**
- This PR hand-picks patches for theme and plugin updates when running `update-releases.py` with the `--overwrite` option

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [ ] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

File: 01 - Community/People/Gru80.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (claremacrae) [inline 01 - Community/People/Gru80.md:1]: This seems to be showing up files that I just reviewed in the previous PR...

For future PRs, this makes me think that it's easier to create a single PR with 2 separate commits - the all-automated bits, and then the manual bits...

**Review by claremacrae — APPROVED** [review]
The changes here look good, so I've clicked Approve, with a suggestion above for future PRs...

File: 01 - Community/People/Gru80.md (hunk 1 above)
Comment (argenos) [reply 01 - Community/People/Gru80.md:1]: yeah, I forgot to say that this is based on the branch for #178. I just forgot to target the right branch.

Comment (claremacrae) [reply 01 - Community/People/Gru80.md]: Thanks, I didn't know that was possible.

**Pull request merged**